src/sequelize.ts

```diff
@@ -51,12 +52,16 @@ export function BigNumberStringType (propName: string): Partial<ModelAttributeCo
   }
 }
 
-export default function (app: Application): void {
+export default async function (app: Application): Promise<void> {
   const sequelize = sequelizeFactory()
   const oldSetup = app.setup
 
   app.set('sequelize', sequelize)
 
+  // Run migration
+  logger.info('Run DB Migrations')
+  await DbMigration.getInstance(sequelize).up()
```

Comment:
You should not run migrations automatically! 
There should be a check here if there are no pending migrations and if so then throw an error, exit and let the user handle this situation with the `db-migration` command!